runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -92,6 +110,16 @@ private void setTags(int itemId, Collection<String> tags)
 
 	boolean findTag(int itemId, String search)
 	{
+		if (search.startsWith("clue"))
+		{
+			loadClue();
```

Comment:
Maybe this should simply return the requirements instead of storing both clue and requirements as field?

Reply:
It's fine to build the reqs every search but I figured why waste the time. I guess it could be changed to just directly compare with each item requirement vs building a list.